test/e2e/watchcache/watchcache_enabled_test.go

```diff
@@ -68,7 +68,7 @@ func TestWatchCacheEnabledForCRD(t *testing.T) {
 	crdClient := apiExtensionsClients.Cluster(cluster).ApiextensionsV1().CustomResourceDefinitions()
 
 	t.Log("Creating wildwest.dev.cowboys CR")
-	wildwest.Create(t, crdClient, cowBoysGR)
+	wildwest.Create(t, logicalcluster.Name{}, crdClient, cowBoysGR)
```

Comment:
cluster, not empty.

Reply:
This is temporary. I'll create a follow up PR to modify clients in `watchcache/` and that will make sure to pass the right cluster name while creating the object.

Reply:
ack